utils/centos7-ci.sh

```diff
@@ -25,9 +25,9 @@ install_dependencies() {
     yum install -y wget tar gcc automake autoconf libtool make unzip \
         curl git which sudo
 
-    # install openresty
+    # install openresty to make rpm work
```

Comment:
Which rpm? It should be our rpm not OpenResty's.

Reply:
fixed